tests/v2prov/defaults/defaults.go

```diff
@@ -3,7 +3,7 @@ package defaults
 import "os"
 
 var (
-	PodTestImage           = "rancher/systemd-node:v0.0.4"
+	PodTestImage           = "rancher/systemd-node:v0.0.5-rc1"
```

Comment:
I've bumped this test image version because the newest version has both amd64 and arm64 images, and we need both for CI.